x-pack/plugins/ml/public/components/controls/select_severity/select_severity.js

```diff
@@ -31,10 +33,26 @@ const optionsMap = {
 };
 
 export const SEVERITY_OPTIONS = [
-  { val: 0, display: 'warning', color: getSeverityColor(0) },
-  { val: 25, display: 'minor', color: getSeverityColor(25) },
-  { val: 50, display: 'major', color: getSeverityColor(50) },
-  { val: 75, display: 'critical', color: getSeverityColor(75) },
+  {
```

Comment:
Now that `SEVERITY_OPTIONS` is being translated, `optionsMap` keys will also need to be translated since now in`onChange` (line 103) `threshold` used to update the service will always be `warning`.
This occurs because in line 107
`const threshold = optionValueToThreshold(optionsMap[valueDisplay]);` 
`optionsMap[valueDisplay]` will always be `undefined` since that translated key will not match any key in `optionsMap`. 
`optionValueToThreshold` will always default to `warning` when threshold is undefined.

Translation can be done inline like 
```
const optionsMap = {
  [i18n.translate('xpack.ml.controls.selectSeverity.warningLabel', { defaultMessage: 'warning' })]: 0,
  [i18n.translate('xpack.ml.controls.selectSeverity.warningLabel', { defaultMessage: 'minor' })]: 25,
  [[i18n.translate('xpack.ml.controls.selectSeverity.warningLabel', { defaultMessage: 'major' })]]: 50,
  [[i18n.translate('xpack.ml.controls.selectSeverity.warningLabel', { defaultMessage: 'critical' })]]: 75,
};
```
or perhaps to prevent duplication of code - translations can be set to a var and used in multiple places.